cyhy_report/customer/generate_report.py

```diff
@@ -183,16 +183,11 @@
 # publicly-accessible network management interfaces that should be protected.
 # This list may change in the future.
 POTENTIAL_NMI_SERVICES = [
-    "bftp",           # FTP
-    "ftp",            # FTP
     "microsoft-ds",   # SMB
     "ms-wbt-server",  # RDP
-    "ni-ftp",         # FTP
-    "rsftp",          # FTP
     "rtelnet",        # Telnet
     "smbdirect",      # SMB
     "telnet",         # Telnet
-    "tftp",           # TFTP
```

Comment:
https://github.com/cisagov/cyhy-system/issues/108 mentions removing the FTP service, which I assumed to mean just regular FTP, but this PR removes a variety of different FTP services (bftp, ftp, ni-ftp, rsftp, tftp).  Can you please confirm that this was the original intent of https://github.com/cisagov/cyhy-system/issues/108?

Reply:
Confirmed with Genevieve, this is correct.

Reply:
OK, thanks for checking on that. 👍